ee/uninstall/uninstall.go

```diff
@@ -0,0 +1,60 @@
+package uninstall
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+	"os"
+
+	"github.com/kolide/launcher/ee/agent"
+	"github.com/kolide/launcher/ee/agent/types"
+)
+
+func Uninstall(ctx context.Context, k types.Knapsack) {
+	uninstallNoExit(ctx, k)
+
+	if err := disableAutoStart(ctx); err != nil {
+		k.Slogger().Log(ctx, slog.LevelError,
+			"disabling auto start",
+			"err", err,
+		)
+	}
+
+	// TODO: remove start up files
+	// TODO: remove installation
```

Comment:
TODO implies we want that. I think it's not clear. Are you planning that?

Reply:
no, removed